modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -27,10 +27,11 @@ const setupArgs = (args: Partial<GetContentArgs>, format: ContentFormat): GetCon
   ...args,
   format,
   get: true,
-  getInner: true
+  getInner: true,
+  content: ''
```

Comment:
This is definitely incorrect as it'll overwrite the content passed in via `args`. As such this should be before the `...args` part so it's a default.

Reply:
After discussion, set `content` is made optional and thus not needed here.